homeassistant/components/brunt/__init__.py

```diff
@@ -1 +1,64 @@
 """The brunt component."""
+from __future__ import annotations
+
+import logging
+
+from aiohttp.client_exceptions import ClientResponseError, ServerDisconnectedError
+from brunt import BruntClientAsync
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+PLATFORMS = ["cover"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
+    """Component setup, run import config flow for each entry in config."""
+    if DOMAIN not in config:
+        return True
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
This is already in `async_setup_entry`.

```suggestion
```

Reply:
Isn't this necessary for the people who have brunt configured though yaml?

Reply:
Nope, if configured in YAML it triggers an import via the config flow, which results in creating an entry. That entry is loaded. The entry setup method below already creates this space.